aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,55 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use anyhow::{Context, Result};
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::missing_errors_doc)]
+pub fn restart(services: &[&ServiceDefinition]) -> Result<()> {
+    // stop all services
+    for service in services.iter().map(|s| s.service) {
+        print!("Stopping {}...", service);
+        let result = Command::new("systemctl")
+            .args(&["stop", service])
+            .output()
+            .context("Failed to call systemctl stop")?;
+
+        if result.status.success() {
+            println!("Stopped!");
+        } else {
+            eprintln!("\nError stopping {}\n", service);
+            io::stdout().write_all(&result.stdout)?;
+            io::stderr().write_all(&result.stderr)?;
+            println!();
```

Comment:
so, uh, did you just not do https://github.com/Azure/iot-identity-service/pull/105#discussion_r573032167 then?

(ditto for the other instances)

Reply:
I did change it to eprintln. Did you mean for me to print the result to stderr ass well?

Reply:
I had meant to include the entire block in my comment.

Basically, what I was proposing was to do something like this:

```rust
eprintln!("\nError stopping {}\n", service);
eprintln!("stdout:")
eprintln!("=======")
io::stdout().write_all(&result.stdout)?;
eprintln!("stderr:")
eprintln!("=======")
io::stdout().write_all(&result.stderr)?;
eprintln!();
```

Reply:
actually, since you've already got an empty `eprintln!` at the end there, might as well split `eprintln!("\nError stopping {}\n", service);` into `eprintln!(); eprintln!("Error stopping {}", service); eprintln!();`